### PR TITLE
Move HistoryOp inside bottom sheet

### LINKED ISSUE #553: [Design] Design improvements to transaction details (HistoryOpScreen)
When a user taps on a transaction cell, a bottom sheet will appear with the transaction details. When the contact initial bubble is tapped on, they are sent to the "account page" (see screenshot): 

Screenshot 2023-12-18 at 4 07 44 PM

Figma link: https://www.figma.com/file/wBBNeEzlsObaE8A8i9iVf7/Mobile-App?type=design&node-id=2521%3A16074&mode=design&t=uXCFjvTSz6zvIGbO-1

### PR DESCRIPTION
**Pull request by wojtus7**
Move HistoryOp screen inside bottom heet on home screen

Closes #553

iOS:

https://github.com/daimo-eth/daimo/assets/42337257/39eb4a5a-a4b7-43c5-9999-c6dd2534272c

Android:

https://github.com/daimo-eth/daimo/assets/42337257/e6993ea3-6540-41eb-92a4-341a978ffedc

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/HomeScreen.tsx [hunk 1]
```diff
@@ -180,7 +197,7 @@ function HomeScreenInner({ account }: { account: Account }) {
             {account.suggestedActions.length > 0 ? (
               <SuggestedActionBox action={account.suggestedActions[0]} />
             ) : (
-              <Spacer h={64} />
+              <Spacer h={screenDimensions.height > 680 ? 64 : 24} />
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/HomeScreen.tsx:200]: which device needs the smaller spacer?

**Review by dcposch — APPROVED** [review]
LGTM, left a question

File: apps/daimo-mobile/src/view/screen/HomeScreen.tsx (hunk 1 above)
Comment (wojtus7) [reply apps/daimo-mobile/src/view/screen/HomeScreen.tsx:200]: My iPhone SE 😖

**Comment by nalinbhardwaj** [comment]
- Remove offline bar on transfer screen:



- Seems like header image related code might have a bug? I get a blank space at the top and the bottom cuts off:



- Pulling up the sheet from the transfer screen also seems bugged. Doesn't pull up at all in the flow default click > pull up:

https://github.com/daimo-eth/daimo/assets/6984346/3e9d3ff0-a564-410c-9d62-a6ba71c69098

- nit: payment link bubble leads to the same page as its transfer currently, can disable tap target for those rows.

- Console warning: Sending  `onAnimatedValueUpdate` with no listeners registered.

- Make sure the interactions work well in pairing with https://github.com/daimo-eth/daimo/pull/610, needs rebase over that.

**Review by nalinbhardwaj — CHANGES_REQUESTED** [review]
see comment, would appreciate if you could test it on a real device as well and make sure it doesn't feel jank in any interactions.

**Comment by wojtus7** [comment]
@nalinbhardwaj I think I resolved all things except:
"Console warning: Sending  onAnimatedValueUpdate with no listeners registered."
But I think this happened also before changes in this PR (?) so I would handle it separately because it can be totally not connected with implemented functionality, but I will surely take a look at it asap

Wanted to let you know but I still need to test it a little more on physical devices to make sure sure everything works smoothly 👌

**Comment by dcposch** [comment]
Rebased on master. (Had to replay using `git diff --patch` due to merge commits.)

Fixed one more issue: when you open a transaction, then swipe down to minimize the bottomsheet again, the transaction didn't go away:
Screenshot 2024-01-20 at 03 51 35

Fixed. Swipe down now returns you to the home HistoryOpList.

**Review by nalinbhardwaj — APPROVED** [review]
lgtm

**Pull request merged**